python/taichi/lang/matrix.py

```diff
@@ -263,45 +247,40 @@ def w(self):
         else:
             return self[3]
 
+    # since Taichi-scope use v.x.assign() instead
```

Comment:
Which function will it invoke if I call `v.x = 233` in Taichi-scope?

Reply:
It's first `v.x.getter` then `.assign()`, see `python/lang/transformer.py`.
```py
In [2]: @ti.kernel 
   ...: def func(x: ti.f32, y: ti.f32): 
   ...:     v = ti.Vector([x, y]) 
   ...:     v.x = 233  # translated into v.x.assign(233)
   ...:     print(v) 
   ...:                                                                                                                                                                                      

In [3]: func(666, 777)                                                                                                                                                                       
[233.000000, 777.000000]
```